README.md

```diff
@@ -270,6 +270,13 @@ If you would like to contribute to the translation of Open Event, you need to [r
 
 Once you have activated your account just proceed to the [translation section](https://hosted.weblate.org/projects/open-event/).
 
+## Pre-commits guide
+
+**Introduction**
+
+Git hook scripts are useful for identifying simple issues before submission to code review. We run our hooks on every commit to automatically point out issues in code such as missing semicolons, trailing whitespace, and debug statements. By pointing these issues out before code review, this allows a code reviewer to focus on the architecture of a change while not wasting time with trivial style nitpicks.
+
+### For installation and configuration, [Click Here](https://pre-commit.com/)
```

Comment:
https://github.com/fossasia/open-event-server/pull/6991#discussion_r425315964.
Plus, I don't think this is a good place in the documentation for the Pre-commits guide. Include it in the development section.